util/cron/test-linux64-gcc47.examples.bash

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+#
+# Test default configuration on examples only, on linux64, with compiler gcc-4.7
+
+CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
+
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="linux64-gcc47.examples"
+
+gcc_setup=/data/cf/chapel/setup_gcc47.bash
+if source "${gcc_setup}"
```

Comment:
Is there a simple way to put all the "load gcc X and error on failure" logic into a common file so in here you would just see something like 

```sh
source $CWD/common-load-gcc.bash
load_gcc(<version>)
```